src/ContentHubClient.php

```diff
@@ -147,8 +151,16 @@ public function __call($method, $args) {
 
       $args = $this->addSearchCriteriaHeader($args);
 
-      return $this->httpClient->__call($method, $args);
+      if (\count($args) < 1) {
```

Comment:
Can we just remove __call? I don't think we need it at all. In related methods, such as get, put, patch, post can be passed to the guzzle http.